tests/test_modules/generated.py

```diff
@@ -93,28 +94,33 @@ class WithUntypedStringDefaultConf:
 class ListValuesConf:
     _target_: str = "tests.test_modules.ListValues"
     lst: List[str] = MISSING
+    lst2: list[str] = MISSING
     enum_lst: List[Color] = MISSING
     passthrough_list: Any = MISSING  # List[LibraryClass]
     dataclass_val: List[User] = MISSING
     def_value: List[str] = field(default_factory=lambda: [])
+    def_value2: list[str] = field(default_factory=lambda: [])
 
 
 @dataclass
 class DictValuesConf:
     _target_: str = "tests.test_modules.DictValues"
     dct: Dict[str, str] = MISSING
+    dct2: dict[str, str] = MISSING
     enum_key: Dict[Color, str] = MISSING
     dataclass_val: Dict[str, User] = MISSING
     passthrough_dict: Any = MISSING  # Dict[str, LibraryClass]
     def_value: Dict[str, str] = field(default_factory=lambda: {})
+    def_value2: dict[str, str] = field(default_factory=lambda: {})
```

Comment:
```suggestion
    def_value2: dict[str, str] = field(default_factory=lambda: {})
```
Is a lambda necessary here? Can't one simply use `dict`?

Reply:
yes, `dict` should work

Reply:
ah, but this is generated code, so I'd rather not mess with it